telluric/collections.py

```diff
@@ -261,9 +261,10 @@ def get_tile(self, x, y, z, sort_by=None, desc=False, bands=None):
         def _get_tiled_feature(feature):
             return feature.get_tiled_feature(x, y, z, bands)
 
-        tiled_features = self.rasters_executor.map(_get_tiled_feature,
-                                                   filtered_fc,
-                                                   timeout=CONCURRENCY_TIMEOUT)
+        with ThreadPoolExecutor(max_workers=MAX_WORKERS) as executer:
```

Comment:
This is creating a `ThreadPoolExecutor` object every time `get_tile` is called. If I understood correctly, an object was added to the class for this purpose:

https://github.com/satellogic/telluric/blob/5c5f70183d78c784a72aff847e19da2f91bc4986/telluric/collections.py#L229

@arielze what do you think? If we are not using `self.rasters_executor` here, I'd rather remove it.

Reply:
I will remove it